### PR TITLE
Fix tag regex in tf file

### LINKED ISSUE #72: TF file for cloud build trigger broken
## Current Behavior
Getting the following error:

```Error: Error parsing .../projects/feast/testing/tf/cloud-build/cloud-build.tf: At 5:176: illegal char escape```

Due to the github tag regex.

### PR DESCRIPTION
**Pull request by zhilingc**
Fixes #72

### CODE REVIEW

File: testing/tf/cloud-build/cloud-build.tf [hunk 1]
```diff
@@ -1,8 +1,12 @@
+variable "base64tagRegex" {
```
Comment (tims) [inline testing/tf/cloud-build/cloud-build.tf]: Wat?

**Comment by tims** [comment]
/hold

I find this base64encoded regex disturbing, can you explain the requirement for it?

**Comment by zhilingc** [comment]
> /hold
> 
> I find this base64encoded regex disturbing, can you explain the requirement for it?

see #72

**Comment by tims** [comment]
it's just unreadable is all.. I was wondering why it's a regex at all and it's just because our build feels weird.

/hold cancel
/lgtm so we can get the build working, but I'm trying to figure out how we make the build more maven native.

**Comment by zhilingc** [comment]
> Think we should just remove the variable and add the regex (in Base64) inline.

Resolved, please have a look

**Pull request merged**